meta-resin-common/recipes-connectivity/bluez5/files/10-local.rules

```diff
@@ -0,0 +1 @@
+ACTION=="add", KERNEL=="hci0", RUN+="/etc/bluetooth/hciconf.sh"
```

Comment:
Could you make the script take an argument as being the device? So we can theoretically support hci1 for example.